packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -109,7 +109,8 @@ class ThemeData {
     unselectedWidgetColor ??= isDark ? Colors.white70 : Colors.black54;
     disabledColor ??= isDark ? Colors.white30 : Colors.black26;
     buttonColor ??= isDark ? primarySwatch[600] : Colors.grey[300];
-    secondaryHeaderColor ??= primarySwatch[50]; // TODO(ianh): dark theme support (https://github.com/flutter/flutter/issues/3370)
+    // Spec doesn't specify a dark theme secondaryHeaderColor, (https://github.com/flutter/flutter/issues/3370)
```

Comment:
if we're going to close the bug, remove the link, and just say something like "dark theme variant is a guess" or something